cocoa/tests_backend/widgets/properties.py

```diff
@@ -23,22 +24,14 @@ def toga_color(color):
         return None
 
 
-@dataclass
-class Font:
-    family: str
-    size: int
-    style: str = NORMAL
-    variant: str = NORMAL
-    weight: str = NORMAL
-
-
 def toga_font(font):
+    traits = font.fontDescriptor.symbolicTraits
     return Font(
-        family={
-            ".AppleSystemUIFont": SYSTEM,
-            "Papyrus": FANTASY,
-        }.get(str(font.familyName), str(font.familyName)),
+        family=str(font.familyName),
```

Comment:
See Q below about this inversion, removing the ".AppleSystemUIFont" normalisation from the platform backend.